enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -535,6 +537,12 @@ func NewPebbleCache(env environment.Env, opts *Options) (*PebbleCache, error) {
 		pebbleOptions.Cache = c
 	}
 
+	desc, err := cdbpebble.Peek(opts.RootDirectory, vfs.Default)
+	if err != nil {
+		return nil, err
+	}
+	created := !desc.Exists
```

Comment:
This variable is supposed to represent a cache that previously didn't exist, and is being created now right? Maybe rename to `newlyCreated`? `created` makes it sound like it existed in the past

Also is another way to implement this without having to import a new package checking if `databaseVersionMetadata` was not in the db? It looks like we return an empty `rfpb.VersionMetadata` in that case. Could we check for an empty lastModifyTime?

Reply:
Older versions of the pebble code did not have the databaseVersionMetadata key so its non-existence does not indicate a new setup. 

Instead of importing the real pebble package, alias the Peek function in our pebble package.

Reply:
Renamed to `newlyCreated` and aliased the `Peek()` call.